Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs

```diff
@@ -19,6 +19,11 @@ public class MixedRealityPointerProfileInspector : BaseMixedRealityToolkitConfig
         private static readonly GUIContent UseEyeTrackingDataContent = new GUIContent("Use Eye Tracking Data");
         private static readonly GUIContent RaycastLayerMaskContent = new GUIContent("Default Raycast LayerMasks");
 
+#if UNITY_2019_3_OR_NEWER
```

Comment:
Could you clarify why this would only apply to Unity 2019 and later?

Reply:
Yep! The Unity API for specifically setting `PlayerSettings.WSACapability.GazeInput` was added in Unity 2019.

Reply:
![image](https://user-images.githubusercontent.com/3580640/163278013-d3d0c7c7-46f3-4554-95e5-c3e33ed5c8a9.png)

https://unity3d.com/unity/whats-new/2019.3.0